app/controllers/idv/hybrid_handoff_controller.rb

```diff
@@ -210,23 +213,26 @@ def confirm_agreement_step_complete
     def confirm_hybrid_handoff_needed
       setup_for_redo if params[:redo]
 
+      idv_session.flow_path = 'standard' if flow_session[:skip_upload_step]
+      # next line temp added for 50/50, remove in future deploy
       flow_session[:flow_path] = 'standard' if flow_session[:skip_upload_step]
+      return if !idv_session.flow_path
```

Comment:
Need to add this back in temporarily.

```suggestion
      # flow_session temp added for 50/50, remove in future deploy.
      return if !idv_session.flow_path && !flow_session[:flow_path]
```